src/Storages/ObjectStorage/StorageObjectStorageCluster.cpp

```diff
@@ -136,4 +260,87 @@ RemoteQueryExecutor::Extension StorageObjectStorageCluster::getTaskIteratorExten
     return RemoteQueryExecutor::Extension{ .task_iterator = std::move(callback) };
 }
 
+std::shared_ptr<StorageObjectStorage> StorageObjectStorageCluster::getPureStorage(ContextPtr context)
```

Comment:
nitpick: nonClusteredStorage instead of pureStorage

Reply:
Just remove method - pure_storage created in constructor anyway, so it always exists.